src/nile/core/account.py

```diff
@@ -297,3 +301,10 @@ def get_counterfactual_address(salt=None, calldata=None, contract="Account"):
         constructor_calldata=calldata,
         deployer_address=0,
     )
+
+
+def set_nile_artifacts_path():
```

Comment:
```suggestion
def get_nile_artifacts_path():
```
A getter with `set` in the name is a little confusing.

Reply:
Haha yeah, I went back and forth on this.